Update logistic_regression.Rmd

pxr687 opened the pull request:
Added a simple matrix illustration of the linear model (thought it fitted well with the text immediately before it).

Changed the name of 'appetite_d' to 'appetite_dummy' - just for readability (less for the reader to hold in mind?).

Added a brief nod to the fact that the odds ratio can be calculated with counts or proportions. (Sometimes proportions are mentioned in the text, sometimes counts, so thought it would be good to mention it explicitly).

Added an illustration (in a dataframe) of how the odds ratio changes as the number of events/nonevents changes.

Added a scatter plot showing how the likelihood is high for good predictions and low for bad predictions.

pxr687 committed:
Update logistic_regression.Rmd

Added a simple matrix illustration of the linear model (thought it fitted well with the text immediately before it).

Changed the name of 'appetite_d' to 'appetite_dummy' - just for readability (less for the reader to hold in mind?).

Added a brief nod to the fact that the odds ratio can be calculated with counts or proportions. (Sometimes proportions are mentioned in the text, sometimes counts, so thought it would be good to mention it explicitly).

Added an illustration (in a dataframe) of how the odds ratio changes as the number of events/nonevents changes.

Added a scatter plot showing how the likelihood is high for good predictions are low for bad predictions.

pxr687 committed:
Update logistic_regression.Rmd

Added a little extra about interpreting odds ratios (interactive cell)

pxr687 committed:
Update logarithms_refreshed.Rmd

Added a quick explanation of why raising to 0 equals 1.

Altered the contents of an array to make the exponent-log reversal more apparent.

pxr687 committed:
Update logarithms_refreshed.Rmd

matthew-brett commented on logarithms_refreshed.Rmd:
```diff
@@ -130,12 +130,28 @@ np.log10(np.sqrt(10))
 
 It is a bit difficult to think about what $10^0$ might mean, but the rule is
-that $10^0$ is equal to 1, and therefore log10 of 1 is 0:
+that $10^0$ is equal to 1. This is the case because the way we calculate divisions like $10^4/10^2$ is by subtracting the bottom exponent from the top exponent e.g. $10^4/10^2 = 10^{4-2} = 10^2$. 
```
This is nice - but maybe this should go at the end when they know about the multiply / add stuff?    Can we just say something like we'll justify this later?  Then put this staff after multiply / add stuff.

I guess we'd also need to say expliclity that dividing is subtracting exponents and give an example like:

```
\frac{10^3}{10^2} = 
\frac{10 * 10 * 10}{10 * 10} = 10 
```

and then your 10^0 example?

pxr687 replied:
I've added a section after the multiply/add section.  I haven't done the subtracting/dividing section yet thought.

Also, sent some suggestions for the logistic regression page itself.

pxr687 committed:
Update logistic_regression.Rmd

Changed `pp_of_label` to `pp_of_correct_label`, as think it might be clearer what these probabilities mean when named this way?

I've added in a little explanation as well. I think the extra breakdown might be helpful?

pxr687 committed:
Update logarithms_refreshed.Rmd
